### ISSUE TITLE
Can't get MapboxDraw control callback called

### ISSUE THREAD

**Issue by Java1Guy** [bbarker/scalajs-mapbox#8]
I am using the MapboxGL interface, creating a Map, creating a MapboxDraw from the mapbox-gl-draw plugin.
All is well except my callback function can't be called.
<img width="635" alt="developer_tools_-_earthview-mapdraw_html" src="https://user-images.githubusercontent.com/69846/34329673-74036e68-e8c6-11e7-8228-3afd86b8d5e5.png">

All in my top level DrawApp:
```
var map = new L.mapbox.MapGL(options)
map.addControl(draw)
map.on("draw.create", updateArea)
```
The callback function:
```
  def updateArea(
//      e: js.Object
    ): Unit = {
    print(">>> updateArea ")
   ...
}
```
The mapbox GL proxy:
```
@js.native
@JSGlobal("mapboxgl.Map")
class MapGL (
            options: js.Object = null
          ) extends js.Object {
  def addControl(control: Control): Map = js.native

  type MapControlCallback = () => Unit
  def on(event: String, handler: MapControlCallback = null): Map = js.native
}
```
I have no clue why the $c_sjsr_AnonFunction1 can't be called :(
I tried changing the function to 
```
var updateArea = () => {....}
```
but no help.

Other than that, this is a very cool project!!
(I'm sure this isn't a scalajs-mapbox project bug directly, but I figure this is where the most applicable knowledge would exist?!)
Basically, I am attempting to reproduce this: https://www.mapbox.com/mapbox-gl-js/example/mapbox-gl-draw/

**Comment by bbarker** [comment]
Hello - first, this looks very cool. I haven't worked with mapbox for some years (never know when I might need to again though), but would welcome any updates to the codebase if you want to make some PRs.

Second - sorry if I'm way off, but it looks like the callback is being accessed from javascript code at a quick glance, which makes me thing you may need to [export](https://www.scala-js.org/doc/interoperability/export-to-javascript.html) it.

Third - Merry Christmas and happy holidays ;-).

**Comment by Java1Guy** [comment]
Changing callback to this didn't help :(
`  @JSExport
  def updateArea(
`
Looks like the BeastNeedsMoreTorque fork has made some of the same changes I have, so might take it up there...

Thanks for checking!
